themes/Metro.theme.xml

```diff
@@ -722,10 +723,12 @@
                         <Grid Margin="0,1,0,0">
                             <Border x:Name="ReadOnlyVisualElement" Background="{StaticResource ReadOnlyBrush}"
                                     Opacity="0"/>
-                            <Grid>
+                            <Grid HorizontalAlignment="{TemplateBinding TextAlignment}">
                                 <ScrollViewer x:Name="ContentElement" BorderThickness="0" IsTabStop="False"
                                               Margin="4,0,2,2" VerticalAlignment="{TemplateBinding VerticalContentAlignment}" Background="{x:Null}"
-                                              HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"/>
+                                              HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}" 
+                                              VerticalScrollBarVisibility="Visible"
+                                              HorizontalScrollBarVisibility="Disabled"/>

```

Comment:
No, the Visible must be set to Auto and not Visible (this must have slipped through somehow)